net/strongswan/Makefile

```diff
@@ -498,6 +498,8 @@ define Package/strongswan/install
 	$(INSTALL_DIR) $(1)/usr/lib/ipsec/plugins
 	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ipsec/libstrongswan.so.* $(1)/usr/lib/ipsec/
 	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ipsec/plugins/libstrongswan-nonce.so $(1)/usr/lib/ipsec/plugins/
+	$(INSTALL_DIR) $(1)/etc/config
+	$(INSTALL_CONF) ./files/ipsec.config $(1)/etc/config/ipsec
```

Comment:
Shouldnt be this set as conffile? :-)

Reply:
Good catch, since it's already defined in `Package/strongswan-swanctl/conffiles`, I moved the commands to `Package/strongswan-swanctl/install`